docs/themes-list/index.rst

```diff
@@ -346,6 +346,19 @@ NWinkler
    :alt:
 
 
+----
+
+.. _oh_my_posh_image:
+
+Oh-My-Posh
+^^^^^^^^^^
+`Theme's page <https://github.com/Bash-it/bash-it/blob/master/docs/themes-list/oh-my-posh.rst>`_
```

Comment:
No need to link it here as we link from the main list

Reply:
I missed that. I'll go look for it. (the RST linter complains no page references it, I didn't check)